charts/base-cluster/templates/monitoring/deadMansSwitch/registration.yaml

```diff
@@ -59,8 +63,8 @@ spec:
                 curl --silent --show-error --fail --retry 5 --max-time 30 "https://hc-ping.com/$PING_KEY/$checkName"
```

Comment:
the line above. I tried to setup myself and it did not work. So I made the API call myself and got the Error 400 Bad Request. After some searching in deeper I received the error that something is incorrect with the channel. So I deleted the channel part from the request and it worked. Maybe I did something wrong in the first place of the configuration or this is a bug

Reply:
an integration in healthchecks.io with the global cluster name must be configured. (According to the documentation)